includes/class-sensei-learner.php

```diff
@@ -62,6 +62,87 @@ public function init() {
 
 		// Try to remove duplicate progress comments to mitigate duplicate enrollment issue.
 		add_action( 'sensei_log_activity_after', [ $this, 'remove_duplicate_progress' ] );
+
+		// Add custom columns.
+		add_filter( 'manage_course_posts_columns', array( $this, 'course_columns' ), 10 );
+		add_action( 'manage_course_posts_custom_column', array( $this, 'course_column_content' ), 10, 2 );
```

Comment:
Yeah, agree. Changed in c5646bd99c8b81e7f383eebffa62cbe913d3ba76. I've added `course` to the name (`add_course_column_heading`) because there could be columns for other post types.